src/selectors/getChildResolvedPath.ts

```diff
@@ -0,0 +1,24 @@
+import { SimplePath, Path } from '../types'
+import { parentOf, head, pathToContext, unroot } from '../util'
+import { State } from '../util/initialState'
+import isContextViewActive from './isContextViewActive'
+
+/** Get thoughtsResolved for a child from it's parent thoughtsResolved.
+ *
+ * @example
+ * // returns ['a', 'b', 'c', 'd']
```

Comment:
Make a note here that the examples use `Contexts` only for clarity; the function actually takes `Paths`.